packages/DockerMicroservice

```diff
@@ -0,0 +1,15 @@
+FROM node AS openactive.microservice
```

Comment:
Noticing this file is still here. Should this have been deleted in this last commit?